Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM ubuntu:focal-20220113
+FROM ubuntu:focal-20220316
 
-ARG VERSION=14.8.2
+ARG VERSION=14.9.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=2.7.5 \
     RUBY_SOURCE_SHA256SUM="2755b900a21235b443bb16dadd9032f784d4a88f143d852bc5d154f22b8781f1" \
     GOLANG_VERSION=1.17.7 \
-    GITLAB_SHELL_VERSION=13.23.2 \
-    GITLAB_PAGES_VERSION=1.54.0 \
-    GITALY_SERVER_VERSION=14.8.2 \
+    GITLAB_SHELL_VERSION=13.24.1 \
```

Comment:
The version seems to be inkonsistent with https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v14.9.0/GITLAB_SHELL_VERSION .